### PR TITLE
Storage check with defined path #154

### PR DESCRIPTION
**Pull request by daniego**
I don't think this change requires any additional tests.

### CODE REVIEW

**Comment by mwarkentin** [comment]
@daniego thanks! Could you update the changelog with a description of the new feature as well?

**Comment by daniego** [comment]
> @daniego thanks! Could you update the changelog with a description of the new feature as well?

That's done!

File: HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,11 @@
 History
 =======
 
+1.0.2 (2020-03-10)
```
Comment (mwarkentin) [inline HISTORY.rst]: Actually this should probably be 1.1.0 - I'll fix that for release.

**Comment by daniego** [comment]
Are you tagging the merge and getting the new package on pypi?

**Comment by mwarkentin** [comment]
@daniego sorry yep I am planning on getting a release out but things have been a little crazier than usual with COVID-19 - our daycares are shut down, etc.

Will try to get it out as soon as I can. :)

**Comment by mwarkentin** [comment]
@daniego if you can test out the release and let me know if everything is looking good from your end that would be helpful for me. :)

**Comment by daniego** [comment]
I released it in our live environment and it works properly!
Thanks

**Comment by mwarkentin** [comment]
Cheers, thanks for letting me know. :)

**Pull request merged**